ci/kind/kind-setup.sh

```diff
@@ -328,7 +328,7 @@ function configure_vlan_subnets {
 function delete_vlan_subnets {
   echo "Deleting VLAN subnets"
 
-  bridge_id=$(docker network inspect $CLUSTER_NAME -f {{.ID}})
+  bridge_id=$(docker network inspect "$CLUSTER_NAME" -f {{.ID}}) || true
```

Comment:
why it appends `|| true`? if the command fails, executing the following code makes sense, right?